XMonad/Actions/TreeSelect.hs

```diff
@@ -161,8 +161,8 @@ import Graphics.X11.Xrender
 -- white       = 0xffffffff
 -- black       = 0xff000000
 -- red         = 0xffff0000
--- blue        = 0xff00ff00
--- green       = 0xff0000ff
+-- blue        = 0xff0000ff
+-- green       = 0xff00ff00
```

Comment:
Can we swap these two lines? :-)

Reply:
Ah. Yeah I don't even know why I changed these.